.template/addons/docker/Dockerfile.tt

```
@@ -108,8 +103,8 @@ RUN rm -rf tmp/docker
 
 <%- if WEB_VARIANT -%>
 # Compile assets
-RUN bundle exec rails i18n:js:export && \
-    bundle exec rails assets:precompile
+RUN SECRET_KEY_BASE=1 DOCKER_BUILD=true bundle exec rails i18n:js:export && \
+    SECRET_KEY_BASE=1 DOCKER_BUILD=true bundle exec rails assets:precompile
```

How about moving that to a Shell script instead? The advantage is that we could document what DOCKER_BUILD is for and why SECRET_KEY_BASE is needed

```suggestion
RUN ./bin/assets-precompile
```